packages/yoroi-ergo-connector/src/inject.js

```diff
@@ -108,6 +126,24 @@ class ErgoAPI {
 
 const ergo = Object.freeze(new ErgoAPI());
 `
+
+const cardanoApiInject = `
+
+// @todo add cardano rpc resolver when it is done
+
+class CardanoAPI {
+    constructor(){
+        this.getInitTimestamp()
+    }
+
+    getInitTimestamp(){
+        return Date.now()
+    }
```

Comment:
@AhmedIbrahim336 This function should always return the same date, when the class instance was created. You need to do something like `this.initTimestamp = Date.now()` in the constructor and then `return this.initTimestamp` inside the function.

Reply:
What are we using the init timestamp for? It's not in the spec. Is this something to show on the UI side? Do we want to do this for Ergo too?

Reply:
it just a place holder for now I think. @vantuz-subhuman  I think you need to answer this

Reply:
Yes